src/index.ejs

```diff
@@ -6,6 +6,24 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 	</head>
 	<body>
+		<nav>
+			<button class="logo" type="button" onclick="window.location = 'https://actionnetwork.org/fundraising/support-lawyers-for-good-government-in-the-fight-to-protect-our-democracy';">
+			</button>
+			<button class="menu" onclick="navToggle(this)">
+  				<div class="bar1"></div>
+  				<div class="bar2"></div>
+  				<div class="bar3"></div>
+			</button>
+		</nav>
 		<div id="timeline"></div>
+		<div class="nav" id="navpane">
+			<ul>
+				<li class = "important"><a href="https://actionnetwork.org/fundraising/support-lawyers-for-good-government-in-the-fight-to-protect-our-democracy">Donate</a></li>
+				<li><a href="http://lawyersforgoodgovernment.org">About</a></li>
+				<li></li>
+			</ul>
+		</div>
 	</body>
-</html>
+	<script>document.getElementById('timeline').style.display = 'absolute'</script>
```

Comment:
I think i put this in there because of the DOM load flicker, but i can remove it